src/main/java/edu/uc/inventorymanager/service/ItemServiceStub.java

```diff
@@ -19,9 +19,7 @@ public void save(Item item) {
 
     @Override
     public Item fetchItemById(int id) {
-        Item item = new Item();
-        item.setDescription("Test Description");
-        item.setId(id);
+        Item item = new Item(id, "Test Description");
```

Comment:
Creating a constructor that does everything in one line reduces technical debt and makes the code cleaner